tv_apps/tv-ftu/js/tv_ftu.js

```diff
@@ -11,35 +13,117 @@ window.addEventListener('load', function() {
       this.APP_USAGE_ENABLE_VALUE = true;
       this.APP_USAGE_DISABLE_VALUE = false;
 
-      this._noButton =
-          document.getElementById('app-usage-no-button');
-      this._yesButton =
-          document.getElementById('app-usage-yes-button');
-      this._terms =
-          document.getElementById('app-usage-terms');
-      this._termsSection =
-          document.getElementById('app-usage-terms-section');
-      this._arrowIcon =
-          document.getElementById('app-usage-arrow-icon');
-
+      this._currentPage = null;
+      this._currentContentSection = null;
+      this._currentContent = null;
+      this._currentArrowIcon = null;
+      this._currentFocused = null;
+      this._currentNavigationTarget = null;
       this.keyNavAdapter = new KeyNavigationAdapter();
-      this.keyNavAdapter.init();
-      this.keyNavAdapter.on('move', this.onKeyMove.bind(this));
-      this.keyNavAdapter.on('enter-keyup', this.onEnterKeyUp.bind(this));
-
-      this.buttonNav = new SimpleKeyNavigation();
-      this.buttonNav.start(
-        [this._noButton, this._yesButton],
-        SimpleKeyNavigation.DIRECTION.HORIZONTAL
-      );
-      this.buttonNav.on('focusChanged', (elem) => {
-        this.fire('focus', elem);
+      this.linkNavigation = new SimpleKeyNavigation();
+
+      this.fteWizard = new FTEWizard('tvFTU.fteWizard');
+      this.fteWizard.init({
+        container: document.body,
+        pageClass: 'page',
+        buttonsClass: 'page-button',
+        propagateKeyEvent: true,
+        ongoNext: this.setupPage.bind(this),
+        ongoPrev: this.setupPage.bind(this),
```

Comment:
I just don't understand, there is only one page on it and you need to add listener to goNext, goPrev, and init? What makes this necessary? I even think you don't have to go with `fteWizard`.